portal_backend/views/permissions.py

```diff
@@ -49,6 +48,7 @@ def has_permission(self, request: Request, view: View) -> bool:
             return True
 
         if user_obj.role == "maintainer":
-            return token_org_slug == request_org_slug
+            if request_org_slug in token_org_slugs:
+                return True
 
         return False
```

Comment:
Since this snippet will only return a boolean, why not directly return the conditions?

```python
return user_obj.role == "maintainer" and request_org_slug in token_org_slugs
```